build/configs/rtl8721csm/hello/defconfig

```diff
@@ -394,6 +394,11 @@ CONFIG_USERMAIN_STACKSIZE=2048
 CONFIG_PTHREAD_STACK_MIN=256
 CONFIG_PTHREAD_STACK_DEFAULT=2048
 
+#
+#SRAM Heap size information
+#
+CONFIG_SRAM_HEAP_SIZE=109
```

Comment:
Please add this to all rtk defconfigs.

Reply:
We will add to other rtk defconfigs once we confirm the configs that needs to be maintained.